cli/src/session.rs

```diff
@@ -469,6 +475,30 @@ impl Session {
             "!configs" => {
                 println!("{:#?}", self.settings);
             }
+            "!options" => {
```

Comment:
seems options is different display with `configs`

Reply:
yes. However, we currently do not have an [options] syntax block. Is it intended to support this content later?